api/src/dev/server/Dockerfile

```diff
@@ -64,8 +65,12 @@ RUN unzip gradle.zip && rm gradle.zip \
 ENV PATH="$PATH:/gradle/bin"
 WORKDIR /
 
+RUN apk add gcc libc-dev g++
+
 RUN gem install --no-document googleauth
 
+RUN gem install rainforest-cli
```

Comment:
FYI: you're likely to need this here as well: https://github.com/all-of-us/workbench/blob/f35b531e374fb40b59d41e7b1f69a93a49ca9083/ci/Dockerfile.circle_build

Doesn't need to be this PR though - should probably wait until you're hooking this up to circle so you can test it.

Reply:
Yeah, I think that should go into a PR for the circle integration.